docs/runtime/runtime-metadata.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "The Metadata of Runtime"
+---
+
+In blockchain, the logic to process transactions and modify storage is usually called STF (State Transition Function). The more popular name among Substrate developers is runtime. 
+
+Substrate module-based metadata provides an overview of blockchain runtime. It helps external code introspect modules, storage items, dispatchable functions and events.
+
+## SRML Metadata
+
+**[Srml-metadata](https://github.com/paritytech/substrate/tree/master/srml/metadata)** is not a "well-known" component of [SRML](runtime/substrate-runtime-module-library.md), since it doesn't provide additional storage entries, dispatchable functions and events. 
+
+It simply provides the data structure to store the overall runtime's metadata, and related codec to convert between raw data and bytes literal. Substrate framework uses the data format called SCALE (Simple Concatenated Aggregate Little-Endian), refer to [Low-level Data Format](overview/low-level-data-format.md) page for more information. 
+
+The structure of the runtime metadata could be changed in the future since Substrate framework is still in rapid evolvement. Here we are using `RuntimeMetadataV4` with Substrate v1.0 branch.
```

Comment:
Applies to everything...

Reply:
The reason that I put such a generic statement here is to show a potential explanation of why we have `V4` or even `V6` in the future.

Reply:
Everything is in development and could be changed in the future.

Reply:
We already have v6.

Reply:
This doc is still based on v1.0, will update based on master later after the content is in good quality.

Reply:
Of course everything is subject to change/development.

Still seems worthwhile to mention that v1.0 is using metadata V4